src/components/AddFood.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function AddFood(props) {
+
+    return (
+        <form onSubmit={props.onAdd}>
```

Comment:
using regular forms and passing the information via the event works fine. However in react, you can also work around the state, having the input values as state elements and updating the input tags dynamically onChange. Then use the state values to pass the information when you call the function.